presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardCleaner.java

```diff
@@ -251,17 +252,15 @@ private void scheduleBackupCleanup()
 
     private void scheduleLocalCleanup()
     {
+        Set<UUID> local = getLocalShards();
+        scheduler.submit(() -> {
+            waitJitterTime();
+            runLocalCleanupAtStartup(local);
+        });
         scheduler.scheduleWithFixedDelay(() -> {
-            try {
-                // jitter to avoid overloading database
-                long interval = this.localCleanerInterval.roundTo(SECONDS);
-                SECONDS.sleep(ThreadLocalRandom.current().nextLong(1, interval));
-            }
-            catch (InterruptedException e) {
-                Thread.currentThread().interrupt();
-            }
+            waitJitterTime();
             runLocalCleanup();
-        }, 0, localCleanerInterval.toMillis(), MILLISECONDS);
+        }, localCleanerInterval.toMillis(), localCleanerInterval.toMillis(), MILLISECONDS);
```

Comment:
Also, think if jitter time parameter needs to be different (smaller) for a cleanup at a startup.